<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QWidget" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设置</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fusion Pixel 8px Monospaced latin</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Kanotale</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>240</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fusion Pixel 12px Monospaced latin</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Fusion Pixel 12px Monospaced latin</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::ContextMenuPolicy::DefaultContextMenu</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox::indicator {border: 1px solid white;}
QCheckBox {spacing: 15px;}
QCheckBox::indicator:checked { background-color: rgb(223, 209, 10);}</string>
   </property>
   <property name="locale">
    <locale language="Chinese" country="China"/>
   </property>
   <property name="text">
    <string> 开启鹿鸣</string>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::AudioVolumeHigh"/>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>190</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Fusion Pixel 12px Monospaced latin</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::ContextMenuPolicy::DefaultContextMenu</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox::indicator {border: 1px solid white;}
QCheckBox {spacing: 15px;}
QCheckBox::indicator:checked { background-color: rgb(223, 209, 10);}</string>
   </property>
   <property name="text">
    <string> 血量锁定</string>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentProperties"/>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>221</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(0, 0, 0);
border: 2px solid white;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>label</zorder>
  <zorder>pushButton</zorder>
  <zorder>checkBox</zorder>
  <zorder>checkBox_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
